mno_analysis_tools/fetch_archives.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+
+from core_data_modules.logging import Logger
+from rapid_pro_tools import RapidProClient
```

Comment:
How is this import possible if you haven't declared it in your Pipfile?

Reply:
I have been using 'update_rapid_pro_client.get_messages' branch to test my implementation